### PR TITLE
Module3_Task2

### PR DESCRIPTION
**Pull request by OlegMazurenko**

### CODE REVIEW

File: Module3/Program.cs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            Task2(data);
+            Console.WriteLine("-----Task3-----");
+            Task3(data);
+            Console.WriteLine("-----Task4-----");
+            Task4(data);
+            Console.WriteLine("-----Task5-----");
+            Task5(data);
+        }
+
+        static void Task1(StoreContext data)
```
Comment (yevhenbilushko) [inline Module3/Program.cs]: good work for make each method for each task
lets move UI  things such as "Console.WriteLine..." out of it
it should have only logic

File: Module3/Program.cs [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        {
+            var products = data.Products.OrderBy(p => p.Name).ToList();
```
Comment (yevhenbilushko) [inline Module3/Program.cs]: no need

Comment (yevhenbilushko) [reply Module3/Program.cs]: i mean this version is ok
var products = data.Products.OrderBy(p => p.Name)

File: Module3/Program.cs [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            foreach(var product in products)
+            {
+                Console.WriteLine($"{product.Name}");
+            }
+        }
+
+        static void Task2(StoreContext data)
+        {
+            var products = data.Products.Join(data.Users,
```
Comment (yevhenbilushko) [inline Module3/Program.cs]: lets add version where u should  Seller info via Product
use Product-Seller relation

File: Module3/Program.cs [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+        static void Task3(StoreContext data)
+        {
+            var categories = data.Categories;
+            var products = data.Products;
+            Console.WriteLine("Category Name | Products Count");
+            foreach (var category in categories)
+            {
+                var productsInCategory = products.Where(x => x.CategoryId == category.Id).ToList();
+                Console.WriteLine($"{category.Name}     |     {productsInCategory.Count}");
```
Comment (yevhenbilushko) [inline Module3/Program.cs]: please make it via LINQ

Comment (OlegMazurenko) [reply Module3/Program.cs]: Didn't i do it via LINQ? Or you mean i should use Join?

Comment (OlegMazurenko) [reply Module3/Program.cs]: Remade it using Join.

Comment (yevhenbilushko) [reply Module3/Program.cs]: great!

**Review by yevhenbilushko — APPROVED** [review]

**Pull request merged**